September Design Update #2

garrying opened the pull request:
Carried over from #130

## Fixes
- [x] Updated Infura logo on `/sponsors`
- [x] Footer links extending beyond viewport
- [x] Updated HTMLproofer implimentation to use a `Rakefile` and 1 week caching to reduce memory failures

## Features
- [x] Add image for _Channel channel, a network for VPN access_ -> a4d59ec
- [x] Add image for _Currency_ -> 3028986
- [x] Add image for _The Magi system_ -> 3028986
- [x] Add description for mesh workshops -> 1061eaa7f8223429e005dcc7283bf28a05c8ab39 and elsewhere
  - [x] NYC Mesh
  - [x] Grant
- [ ] refine zine library page
  - [ ] form refinement (mailing address, note formatting (use of small), deadline (open call until conf)) @thinkbulecount2 
  - [x] adding graphics @garrying 
- [ ] add content on environment @dcwalk
  - [x] under `/about` -> first pass: https://github.com/ournetworks/ournetworks.ca/pull/133/commits/99dcad74697aaa389313120d148f58f3a563e714
  - [ ] add mention of Global Climate Strike on program
- [x] Add kick-off details (performances) -> d4f272a
- [x] Add keynote -> https://github.com/ournetworks/ournetworks.ca/pull/133/commits/6cbc55a8f71f5b73d47392f0e3c0e97e736aa099
  - [ ] Add image?
- [x] Mention IPFS hash on site @benhylau @garrying 
  - Note: There's an issue with `ipfs-deploy` where it outputs the incorrect hash. Pausing for now -> 97b8befecd4b1325875e85d665abe0742cf2549d.
  - Tracking the incorrect hash Issue here: https://github.com/ipfs-shipyard/ipfs-deploy/issues/55
  - Manual pin to Pinata atm
- [x] Added new `.overflow-wrap` class for keeping long lines in viewport

dcwalk commented:
Did first pass on Environment page, I'm not sure if the format is correct. 

Also @garrying can you say more about the IPFS-deploy issue, is there an issue in that repo we can point to/track?

benhylau commented on _includes/footer.html:
```diff
@@ -9,6 +9,9 @@
         &gt;&gt;&gt; Dat > <a href="dat://eae17983cd6833287ab10ff4c1a3ca5d9ae7af7049eaa525912950720ece2e49/">dat://eae17983cd6833287ab10ff4c1a3ca5d9ae7af7049eaa525912950720ece2e49/</a>
       </p>
+      <p class="mb-0">
+        &gt;&gt;&gt; IPFS > <a href="https://ipfs.io/ipns/ipfs-website.ournetworks.ca/">https://ipfs.io/ipns/ipfs-website.ournetworks.ca/</a>
```
Link `ipfs.infura.io`? ipfs.io is really unreliable.

garrying replied:
Looks like DNSlink and IPNS isn't supported on infura yet. Changing this over to the hash for now: https://github.com/ournetworks/ournetworks.ca/pull/133/commits/538f86da8452d74dbb4b04c1d3471e680b0d65b2#diff-7693eb643e8fe8b8d46b407909b408ceR13

benhylau commented:
Can we alter the way the DAT (or IPFS) hash is presented so it doesn't break mobile layout?

garrying commented:
> Also @garrying can you say more about the IPFS-deploy issue, is there an issue in that repo we can point to/track?

I've updated the description with a link to the issue

> Can we alter the way the DAT (or IPFS) hash is presented so it doesn't break mobile layout?

Added `.overflow-wrap` in 538f86d

garrying commented on environment.md:
```diff
@@ -0,0 +1,23 @@
+parent: "About the event"
+---
+
+Acknowledging both questions of what goes into sustaining an event as well as the layered and uneven climate crisis we are currently experiencing, as organizers we felt it was important to state our commitments to addressing these entwined issues.
```
Feels like this could be two sentences?

dcwalk commented:
Okay did my pass at adding content I was responsible for! I think there is more (allllways), but not for this PR
